packages/cli/src/core/__tests__/findPlugins-test.js

```diff
@@ -8,8 +8,9 @@
  * @emails oncall+javascript_foundation
  */
 
+import findPlugins from '../findPlugins';
+
 const path = require('path');
```

Comment:
Any reason `path` is required, not imported? Same in other files, where we use `fs`. Is that for mocking functionality?

Reply:
There are places that there's a mock before the require. In this case, I believe the import may fail (had this issue in the past). I'll check anyway.

Reply:
`jest.mock` and `jest.unmock` are hoisted above imports with `babel-jest` by default by Jest so that's not an issue

Reply:
But in some places it caused errors. For example in `getPlatforms` we have:

```
const builtInPlatforms = {
  ios: require('./ios'),
  android: require('./android'),
};
```

If I change it to imports, many errors pop.
Of course, I can give extra effort in order to filter _all_ this cases and try to fix the tests to work as expected.

Reply:
Nah, that's not necessary. `require`s are OK in tests when they make testing easier (e.g. when you want to reset modules in between tests in a describe block).

Reply:
In this case I believe we're done